cypress/integration/rendering/classDiagram.spec.js

```diff
@@ -423,4 +423,67 @@ describe('Class diagram', () => {
     );
     cy.get('svg');
   });
+
+  it('20: should render class diagram with newlines in title', () => {
```

Comment:
You can remove the numberings from the tests. It was an old format and no one's sure why it was added. 

Also, can you add tests for string labels? Or is it already present in the file?

```
classDiagram
    class Animal["Animal with a\nNewline"]
```

Reply:
Should I also remove the two commented tests?